.circleci/config.yml

```diff
@@ -1,29 +1,14 @@
 version: 2.1
 orbs:
-  architect: giantswarm/architect@0.8.11
-
-jobs:
-  build:
-    machine: true
-    steps:
-    - checkout
-
-    - run: |
-        wget -q $(curl -sS -H "Authorization: token $RELEASE_TOKEN" https://api.github.com/repos/giantswarm/architect/releases/tags/v1.0.0 | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
-        chmod +x ./architect
-        ./architect version
-    - run: ./architect build
-    - store_test_results:
-        path: /tmp/results
-    - persist_to_workspace:
-        root: .
-        paths:
-        - ./chart-operator
+  architect: giantswarm/architect@0.8.12
 
 workflows:
   build:
     jobs:
-      - build:
+      - architect/go-build:
+          name: go-build
+          binary: chart-operator
+          resource_class: large
```

Comment:
Isn't medium enough here?

```suggestion
```

Reply:
Yes probably but I'd like to see the speed difference. With large it takes 01:26.

Reply:
Time is the same. I'll also downgrade app-operator. 👍